docs/source/conf.py

```diff
@@ -43,20 +46,35 @@
     "sphinx.ext.autosummary",  # Create API doc summary texts from the docstrings.
     "sphinx.ext.inheritance_diagram",  # For class inheritance diagrams (see coregistration.rst).
     "sphinx_autodoc_typehints",  # Include type hints in the API documentation.
-    "sphinxcontrib.programoutput"
+    "sphinxcontrib.programoutput",
+    "sphinx_gallery.gen_gallery",  # Examples gallery
+    "sphinx.ext.intersphinx",
 ]
 
-autosummary_generate = True
+#autosummary_generate = True
+
+intersphinx_mapping = {
+    "geoutils": ("https://geoutils.readthedocs.io/en/latest", None),
+    "numpy": ("https://numpy.org/doc/stable", None),
+    "matplotlib": ("https://matplotlib.org/stable", None),
+}
+
+sphinx_gallery_conf = {
```

Comment:
reads like that wasn't easy :D

Reply:
Nope hahaha!

Reply:
So that's used to link to the documentation of these packages? Great!!